src/js/leaflet.storage.xhr.js

```diff
@@ -66,9 +68,10 @@ L.Storage.Xhr = {
                     L.Storage.fire('ui:alert', {content: msg, level: 'error', duration: 100000, actions: actions});
                 }
                 else {
-                    if (xhr.status !== 0) {  // 0 === request cut by user
```

Comment:
Removing this line, you are removing a needed check: when a use kill a request, by hitting ESC or by changing page while an ajax is ongoing, we don't want to have an error to appear on the page, because it's not an error.